### PR TITLE
Move composer binaries to dev/bin directory.

### PR DESCRIPTION
**Pull request by rdoursenaud**
Since we used to remove them to not clutter the htdocs/includes dir
and these are only useful for developers.

https://getcomposer.org/doc/articles/vendor-binaries.md#can-vendor-binaries-be-installed-somewhere-other-than-vendor-bin-

### CODE REVIEW

**Comment by simnandez** [comment]
@rdoursenaud This should be in the Develop branch. 4.0 is freeze, nd this is not really a bug... What you think?

File: dev/bin/parallel-lint [hunk 1]
```diff
@@ -0,0 +1 @@
+../../htdocs/includes/jakub-onderka/php-parallel-lint/parallel-lint
```
Comment (eldy) [inline dev/bin/parallel-lint:1]: What is goal of having a file dev/bin/parallel-lint with one line that point to the runtime libraries dir "includes" of dolibarr ?

**Comment by rdoursenaud** [comment]
@simnandez I believe it's frozen for user features. This is a developer feature and is a needed milestone for me to add end-to-end (aka functional) testing into the C.I. without too much hassle.
No user visible change.
I think that more testing could be appreciated in a currently supported release.
@eldy Not my design… ask upstream ;)

**Comment by eldy** [comment]
parallel-lint no more required.

**Pull request closed without merging**